apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -34,7 +35,9 @@ public sealed class SpannerCommandTextBuilder
         private const string AlterCommand = "ALTER";
         private const string CreateCommand = "CREATE";
         private const string DropCommand = "DROP";
+        //TODO(benwu): verify if whitespace between CREATE/DROP and DB is allowed.
         private const string CreateDatabaseCommand = "CREATE DATABASE";
+        private const string DropDatabaseCommand = "DROP DATABASE";
```

Comment:
Should you have a space after this? At the moment it *looks* like a DDL of `DROP DATABASEFOO` will have the same effect as `DROP DATABASE FOO` and presumably it shouldn't.

(Ditto CREATE)